src/client.ts

```diff
@@ -6408,11 +6408,11 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
      * @return {Promise<string[]>} Resolves to a set of rooms
      * @return {module:http-api.MatrixError} Rejects: with an error response.
      */
-    public async _unstable_getSharedRooms(userId: string): Promise<string[]> { // eslint-disable-line
+    public async _unstable_getMutualRooms(userId: string): Promise<string[]> { // eslint-disable-line
```

Comment:
I'm not seeing the benefit of the function rename for this change. Even with the fallback semantics implied by the server-side feature flags, we don't need to rename this function.